{
	"document_title": "Security update for python3-lxml",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE-SU-2022:3934-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2022-11-10T12:15:10Z",
				"description": "current"
			}
		},
		"initial_release_date": "2022-11-10T12:15:10Z",
		"current_release_date": "2022-11-10T12:15:10Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "Security update for python3-lxml",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "This update for python3-lxml fixes the following issues:\n\n  - CVE-2021-28957: Fixed XSS due to missing input sanitization for HTML5 attributes (bsc#1184177). \n\n",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "SUSE-2022-3934,SUSE-SLE-Module-Public-Cloud-Unrestricted-12-2022-3934",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/update/announcement/2022/suse-su-20223934-1/",
			"description": "Link for SUSE-SU-2022:3934-1"
		},
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2022-November/012872.html",
			"description": "E-Mail link for SUSE-SU-2022:3934-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1184177",
			"description": "SUSE Bug 1184177"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2021-28957/",
			"description": "SUSE CVE CVE-2021-28957 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "SUSE Linux Enterprise Module for Public Cloud 12",
				"branch": [
					{
						"type": "Product Name",
						"name": "SUSE Linux Enterprise Module for Public Cloud 12",
						"full_product_name": {
							"text": "SUSE Linux Enterprise Module for Public Cloud 12",
							"product_id": "SUSE Linux Enterprise Module for Public Cloud 12",
							"cpe": "cpe:/o:suse:sle-module-public-cloud:12"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python3-lxml-3.3.5-3.15.1",
				"full_product_name": {
					"text": "python3-lxml-3.3.5-3.15.1",
					"product_id": "python3-lxml-3.3.5-3.15.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python3-lxml-doc-3.3.5-3.15.1",
				"full_product_name": {
					"text": "python3-lxml-doc-3.3.5-3.15.1",
					"product_id": "python3-lxml-doc-3.3.5-3.15.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python3-lxml-3.3.5-3.15.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Public Cloud 12"
			},
			{
				"product_reference": "python3-lxml-doc-3.3.5-3.15.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Public Cloud 12"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "An XSS vulnerability was discovered in python-lxml's clean module versions before 4.6.3. When disabling the safe_attrs_only and forms arguments, the Cleaner class does not remove the formaction attribute allowing for JS to bypass the sanitizer. A remote attacker could exploit this flaw to run arbitrary JS code on users who interact with incorrectly sanitized HTML. This issue is patched in lxml 4.6.3.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2021-28957",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"SUSE Linux Enterprise Module for Public Cloud 12:python3-lxml-3.3.5-3.15.1",
						"SUSE Linux Enterprise Module for Public Cloud 12:python3-lxml-doc-3.3.5-3.15.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "4.3",
					"vector_v_2": "AV:N/AC:M/Au:N/C:N/I:P/A:N"
				},
				"score_set_v_3": {
					"base_score_v_3": "6.1",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					},
					"url": "https://www.suse.com/support/update/announcement/2022/suse-su-20223934-1/"
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2021-28957.html",
					"description": "CVE-2021-28957"
				},
				{
					"url": "https://bugzilla.suse.com/1184177",
					"description": "SUSE Bug 1184177"
				}
			]
		}
	]
}
